micro-ui/web/micro-ui-internals/packages/modules/pgr/src/pages/employee/ComplaintDetails.js

```
@@ -0,0 +1,505 @@
+                  })}
+              </ConnectingCheckPoints>
+            )}
+          </React.Fragment>
+        )}
+      </Card>
+      {fullscreen ? (
+        <PopUp>
+          <div className="popup-module">
+            <HeaderBar main={<Heading label="Complaint Geolocation" />} end={<CloseBtn onClick={() => close(fullscreen)} />} />
+            <div className="popup-module-main">
+              <img src="https://via.placeholder.com/912x568" />
```

Biome

[error] Provide a text alternative through the alt, aria-label or aria-labelledby attribute

Meaningful alternative text on elements helps users relying on screen readers to understand content's purpose within a page.

(lint/a11y/useAltText)